### PR TITLE
feat - added item listing page

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Used Responsive grid
-> Used Card of my component library
-> Tried to make ii responsive

This is the preview of page...
https://deploy-preview-3--music-mart-screens.netlify.app/listing-page/listing-page.html

### CODE REVIEW

File: base.css [hunk 1]
```diff
@@ -94,6 +103,24 @@ button {
   align-items: flex-start;
 }
 
+.ali-ce {
+  align-items: center;
+}
```
Comment (devananth) [inline base.css:108]: Naming can be improved. `y-center` would make sense , else `align-center`

File: base.css [hunk 2]
```diff
@@ -94,6 +103,24 @@ button {
   align-items: flex-start;
 }
```
Comment (devananth) [inline base.css:104]: 1.Extra lines can be removed
2. `align-items` is repeated.

**Review by devananth — CHANGES_REQUESTED** [review]
Suggested few changes. 

Code seems to good. Good Work ! 

One thing , you can have the images in a `assets` folder rather than specifing the `image url`.

File: base.css (hunk 2 above)
Comment (ashishnnnnn) [reply base.css:104]: I want to have 3-4 categories of align-items like flex-start,centre etc.,
That's why repeated

File: base.css (hunk 1 above)
Comment (ashishnnnnn) [reply base.css:108]: As there are 3-4 categories of align-items.. to make sure i understand each of them .. i followed this naming convention

File: base.css (hunk 2 above)
Comment (ashishnnnnn) [reply base.css:104]: I want to have 3-4 categories of align-items like flex-start,centre etc.,
That's why repeated

File: base.css (hunk 1 above)
Comment (ashishnnnnn) [reply base.css:108]: As there are 3-4 categories of align-items.. to make sure i understand each of them .. i followed this naming convention

**Comment by ashishnnnnn** [comment]
> Suggested few changes.
> 
> Code seems to good. Good Work !
> 
> One thing , you can have the images in a `assets` folder rather than specifing the `image url`.

**Review by SukanyaSen530 — APPROVED** [review]

**Pull request merged**